src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Properties/InterceptedProjectProperties/BuildPropertyPage/DefineConstantsCSharpValueProvider.cs

```diff
@@ -7,25 +7,28 @@ namespace Microsoft.VisualStudio.ProjectSystem.Properties;
 
 [ExportInterceptingPropertyValueProvider(ConfiguredBrowseObject.DefineConstantsProperty, ExportInterceptingPropertyValueProviderFile.ProjectFile)]
 [AppliesTo(ProjectCapability.CSharpOrFSharp)]
-internal class DefineConstantsValueProvider : InterceptingPropertyValueProviderBase
+internal class DefineConstantsCSharpValueProvider : InterceptingPropertyValueProviderBase
```

Comment:
This is also used for F# (according to `AppliesTo` on the line above) so the new name is potentially misleading.

Reply:
renamed to DefineConstantsCAndFSharpValueProvider